go.mod

```diff
@@ -8,8 +8,8 @@ require (
 	github.com/denisbrodbeck/machineid v1.0.1
 	github.com/ebfe/scard v0.0.0-20190212122703-c3d1b1916a95
 	github.com/gorilla/websocket v1.4.2
-	github.com/grandcat/zeroconf v1.0.0
 	github.com/kardianos/service v1.2.0
+	github.com/libp2p/zeroconf/v2 v2.2.0
```

Comment:
I didn't try a newer version of `grandcat/zeroconf`.
We could consider trying as well, though the `libp2p` version seems to work quite well. I've noticed some improvements in robustness of mDNS discovery since switching to it, though I'm not sure if it's because of the changes in the library or refactoring our own code.

There are also quite a few projects using `grandcat/zeroconf`, some of them are quite popular:
https://github.com/grandcat/zeroconf/network/dependents

It probably does have a larger community around it.

Looking at the difference between `grandcat/zeroconf` and `libp2p/zeroconf`:

https://github.com/libp2p/zeroconf/compare/master...grandcat%3Azeroconf%3Amaster
https://github.com/grandcat/zeroconf/compare/master...libp2p:zeroconf:master

There's not a lot in `grandcat/zeroconf` that's not in `libp2p/zeroconf`, but the latter seems to contain quite a few fixes. It's also easier to use because of an API change described [here](https://github.com/grandcat/zeroconf/issues/98), which was implemented in the libp2p version.

Reply:
I tested a newer version of `grandcat/zeroconf` and it seems to work just as well.

Reply:
I'd say let's stick with the fork since we've been testing with it and we know it works well.  I'll add a comment to `go.mod`.

Reply:
[Add notes about libp2p/zeroconf dependency choice](https://github.com/dividat/driver/pull/127/commits/ad56fd209c664e2b0fc8ea9fb71bcb06ada10d17)